assignments/prototypes.js

```diff
@@ -23,6 +49,35 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+
+const CharacterStats = function(obj){
+  this.createdAt = obj.createdAt,
+  this.name = obj.name,
+  this.dimensions = obj.dimensions,
```

Comment:
if you use: 
`GameObject.call(this, obj);`
 you don't have to type those three properties again.

Review: 
https://learn.lambdaschool.com/fsw/module/rec0awunlezbpit7m

> Let’s look at how inheritance works with prototypes. Here we create a Child constructor. Notice we are using the call() method to bind this to Person.
> 
> function Child(childAttributes) {
>   Person.call(this, childAttributes); // binding this to Person
>   this.isChild = childAttributes.isChild; // this will be a special attribute to Child